src/sprites.py

```diff
@@ -241,23 +295,25 @@ def get_state(self):
         self.state = 'walk' if self.direction else 'idle'
 
     def get_facing_direction(self):
-        # prioritizes vertical animations, flip if statements to get horizontal ones
+        # prioritizes vertical animations, flip if statements to get horizontal
+        # ones
         if self.direction.x:
             self.facing_direction = 'right' if self.direction.x > 0 else 'left'
         if self.direction.y:
             self.facing_direction = 'down' if self.direction.y > 0 else 'up'
 
     def get_target_pos(self):
-        vectors = {'left': pygame.Vector2(-1, 0), 'right': pygame.Vector2(1, 0), 'down': pygame.Vector2(0, 1),
-                   'up': pygame.Vector2(0, -1), }
+        vectors = {'left': pygame.Vector2(-1, 0), 'right': pygame.Vector2(
+            1, 0), 'down': pygame.Vector2(0, 1), 'up': pygame.Vector2(0, -1), }
```

Comment:
Also here, I wonder why the formatter doesn't put one single key-value per line?

Reply:
Some of these are from autopep8, and do look a little weird. Good catch, I'll fix these.